_alerts/2017-07-12-potential-split.md

```diff
@@ -0,0 +1,105 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain split"
+shorturl: "potential-split"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use starting July 31st (click here for details)"
+
+## 7/12 to 7/28: "warning" (orange); 7/29 until resolution: "alert" (red)
+bannerclass: "warning"
+---
+{% assign start='<span class="date">2017/08/01 00:00 UTC</span>' %}
+
+*Last updated: <span class="date">2017/07/12 12:00 UTC</span>.  This
+page will be updated when new information becomes available.  See the
+[list of updates][].*
+
+At {{start}}, Bitcoin confirmation scores may become unreliable for
```

Comment:
We should also add a few sentences at the very beginning about what a chain split is and what bitcoin confirmation scores are. This will be really confusing to people who don't know.